core/controllers/contributor_dashboard.py

```diff
@@ -285,7 +285,9 @@ def get(self):
                 contribution_rights.can_review_questions
                 if contribution_rights else False),
             'can_suggest_questions': (
-                config_domain.CONTRIBUTOR_CAN_SUGGEST_QUESTIONS.value)
+                config_domain.CONTRIBUTOR_CAN_SUGGEST_QUESTIONS.value and
```

Comment:
Are we going to enable this flag by default? (I would suggest either remove this flag or have this as "False" by default.)